src/EnergyPlus/ElectricBaseboardRadiator.cc

```diff
@@ -905,7 +906,12 @@ namespace ElectricBaseboardRadiator {
         Real64 constexpr SmallestArea(0.001); // Smallest area in meters squared (to avoid a divide by zero)
 
         // Initialize arrays
-        state.dataHeatBalFanSys->SurfQElecBaseboard = 0.0;
+        for (auto &elecBaseboard : state.dataElectBaseboardRad->ElecBaseboard) {
+            for (int radSurfNum = 1; radSurfNum <= elecBaseboard.TotSurfToDistrib; ++radSurfNum) {
+                int surfNum = elecBaseboard.SurfacePtr(radSurfNum);
+                state.dataHeatBalFanSys->surfQRadFromHVAC(surfNum).ElecBaseboard = 0.0;
```

Comment:
Instead of zeroing this for all surfaces, only need to touch the surfaces that actually receive radiant heat from electric baseboards. Similar changes for the other four equipment types (chilled panel, HW baseboard, Steam baseboard, high temp radiant).